regulations/tasks.py

```diff
@@ -57,10 +62,26 @@ def publish_metadata(response, key):
     )
 
 
-def build_comment(body):
+def comment_to_markdown(body):
     return loader.render_to_string('regulations/comment.md', body)
 
 
+@contextlib.contextmanager
+def html_to_pdf(html):
+    try:
+        path = tempfile.mkdtemp()
+        html_path = os.path.join(path, 'document.html')
+        pdf_path = os.path.join(path, 'document.pdf')
+        with open(html_path, 'w') as fp:
+            fp.write(html)
+        subprocess.check_output(['wkhtmltopdf', html_path, pdf_path])
+        pdf_file = open(pdf_path)
+        yield pdf_file
+    finally:
+        pdf_file.close()
+        shutil.rmtree(path)
```

Comment:
:+1: 

Now that you mention it, there's a similar issue in `build_attachments`--if we die in the comprehension over files, we lose references to the file pointers and error on trying to close them.